js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/EvaluationDetailDialog.tsx

```diff
@@ -348,3 +401,25 @@ const EvaluationDetailDialogStyle = {
   minHeight: '400px',
   maxHeight: '1400px',
 };
+
+const BackButton = styled.div`
+  width: 25px;
+  height: 30px;
+  border: 1px solid ${Colors.keylineDefault()};
+  border-radius: 3px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: calc(50% - 15px);
+  bottom: calc(50% - 15px);
+  background: ${Colors.backgroundDefault()};
+  z-index: 10;
+
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  &:hover {
+    background: ${Colors.backgroundDefaultHover()};
+  }
+`;
```

Comment:
Can this just be an actual `Button`? (We should avoid using non-interactive elements like div or span for interactive UI.)